dangerfile.ts

```diff
@@ -82,8 +81,8 @@ if (testFilesThatDontExist.length > 0) {
 // A component should have a corresponding story reference, so that we're consistent
 // with how the web create their components
 
-const reactComponentForPath = (path) => {
-  const content = fs.readFileSync(path)
+const reactComponentForPath = (filePath) => {
+  const content = fs.readFileSync(filePath).toString()
```

Comment:
I usually tell `readFileSync` the encoding as `utf-8`, in which case you get back a string immediately.